aioredis/connection.py

```diff
@@ -107,8 +107,8 @@ def __init__(self, reader, writer, *, address, encoding=None, loop=None):
         self._address = address
         self._loop = loop
         self._waiters = deque()
-        self._parser = hiredis.Reader(protocolError=ProtocolError,
-                                      replyError=ReplyError)
+        self._parser = Reader(protocolError=ProtocolError,
```

Comment:
Yes, I will add an argument for parser class.
However hiredis will still be a requirement and default parser.
I've added PyReader to make lib run on PyPy3.